src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    class="top__bar__search"
+    data-qa="big"
+    >
+      <img
+        src="/images/Search.svg"
+        alt="Search Logo"
+        class="search__img__big"
+      >
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="search__form"
+      >
+    </form>
+
+    <form
+    action="#"
+    method="post"
+    class="top__bar__search top__bar__search--small"
+    data-qa="small"
+    >
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      class="top__bar__search top__bar__search--small"
      data-qa="small"
    >
```
Consider fixing indents according to the code style guide